src/extractor/guidance/turn_handler.cpp

```diff
@@ -705,9 +705,8 @@ void TurnHandler::handleDistinctConflict(const EdgeID via_edge,
         const auto left_classification = node_based_graph.GetEdgeData(left.eid).road_classification;
         const auto right_classification =
             node_based_graph.GetEdgeData(right.eid).road_classification;
-        if (canBeSeenAsFork(left_classification, right_classification))
-            assignFork(via_edge, left, right);
-        else if (left_classification.GetPriority() > right_classification.GetPriority())
+
```

Comment:
The c&p issue above I get but why do we need the change here removing the `canBeSeenAsFork` branch?